src/build.props

```diff
@@ -1,8 +1,27 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!--
+  By convention, any project whose name starts with "Test." is a test project. But there are
+  a couple of projects which, although _related_ to testing, do not actually contain tests.
+  These projects set IsTestProject to false before importing build.props. Do not apply the
+  convention to those projects.
+  -->
+  <PropertyGroup Condition="'$(IsTestProject)' == ''">
+    <IsTestProject>False</IsTestProject>
+    <IsTestProject Condition="$(MSBuildProjectName.StartsWith('Test.'))">True</IsTestProject>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsExeProject>False</IsExeProject>
+    <IsExeProject Condition="'$(OutputType)' == 'Exe'">True</IsExeProject>
+
+    <TargetFrameworks Condition="$(IsExeProject)">netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
```

Comment:
>netcoreapp2 [](start = 64, length = 11)

Why do we have both netcoreapp2.1 and netcoreapp3.1? can we simply make do with netcoreapp3.1?

Reply:
Agreed. Also agree with your offline comment that we can defer.

---
In reply to: [476587808](https://github.com/microsoft/sarif-sdk/pull/2032#discussion_r476587808) [](ancestors = 476587808)

Reply:
Will add this to the infra-debt work item for removing `net45` support from `Sarif.Sdk` package.

---
In reply to: [476590516](https://github.com/microsoft/sarif-sdk/pull/2032#discussion_r476590516) [](ancestors = 476590516,476587808)